src/site/asciidoc/manual/json-template-layout.adoc.vm

```diff
@@ -476,20 +499,35 @@ Resolve the stack trace into a string field:
 {
   "$resolver": "exception",
   "field": "stackTrace",
-  "stringified": true
+  "stackTrace": {
+    "stringified": true
+  }
 }
 ----
 
-| exceptionRootCause
-| identical to `exception` resolver
-a|
-Resolves the fields of the innermost `Throwable` returned by
-`logEvent.getThrown()`.
+Resolve the stack trace into a string field such that the content will be
+truncated by the given points:
 
-Note that this resolver is toggled by
-`log4j.layout.jsonTemplate.stackTraceEnabled` property.
+[source,json]
+----
+{
+  "$resolver": "exception",
+  "field": "stackTrace",
+  "stackTrace": {
+    "stringified": true,
+    "truncatedStringSuffix": ">",
+    "truncationPointStrings": ["at javax.servlet.http.HttpServlet.service"]
```

Comment:
I replied to the dev list but the formatting was bad so I'm following up here. Looks like a typo:

```
"truncationPointMatcherStrings": ["at javax.servlet.http.HttpServlet.service"]
```

Reply:
Sharp catch! Fixed.